File: Inputs.java
import java.util.Scanner;

public class Inputs {
    public static String GetString(String prompt) {
        Scanner input = new Scanner(System.in);
        System.out.print(prompt);
        String str_res = input.next();

        if (str_res.equals(null)) {
            System.err.println("Inputs.GetString(): String input cannot be null.");
        }

        return str_res;
    }

    public static int GetInt(String prompt) {
        Scanner input = new Scanner(System.in);
        System.out.print(prompt);
        int int_res = input.nextInt();

        return int_res;
    }
}

File: Inputs.cs
public class Inputs {
    
    public static string GetString(string prompt) {
        Console.Write(prompt);
        
        string str_res = Console.ReadLine();

        if (str_res is null) {
            Console.WriteLine("Inputs.GetString(): String input cannot be null.");
        }
        
        return str_res;

    }

    public static int GetInt(string prompt) {
        Console.Write(prompt);
        int int_res = Convert.ToInt32(Console.ReadLine());

        return int_res;
    }
}

